src/SosThreadingTools/SosThreadingTools.csproj

```diff
@@ -1,15 +1,13 @@
 ﻿<Project Sdk="MSBuild.Sdk.Extras">
   <PropertyGroup>
-    <TargetFrameworks>net472</TargetFrameworks>
+    <TargetFrameworks>net7.0-windows</TargetFrameworks>
     <RuntimeIdentifiers>win-x86;win-x64</RuntimeIdentifiers>
     <ExtrasBuildEachRuntimeIdentifier>true</ExtrasBuildEachRuntimeIdentifier>
+    <EnableDynamicLoading>true</EnableDynamicLoading>
+    <AllowUnsafeBlocks>true</AllowUnsafeBlocks>
 
     <AssemblyName>$(MSBuildProjectName)</AssemblyName>
```

Comment:
You might want to change this to `<AssemblyName>$(MSBuildProjectName)Impl</AssemblyName>` and then create `<DnneNativeBinaryName>$(MSBuildProjectName)</DnneNativeBinaryName>` for the native binary.

Reply:
Agreed.